rm/rm-server/src/test/java/org/ow2/proactive/resourcemanager/selection/statistics/ProbabilisticSelectionManagerTest.java

```diff
@@ -252,12 +252,11 @@ public void testVariableBindings() throws Exception {
         ManagerObjects managerObjects = new ManagerObjects(1).invoke();
         SelectionManager selectionManager = managerObjects.getSelectionManager();
         ArrayList<RMNode> freeNodes = managerObjects.getFreeNodes();
-        Map<String, Serializable> bindings = Collections.singletonMap("TOTO", (Serializable) "value");
+
```

Comment:
I think you should add one more check, for example when bindings contain a map with  two keys, we want to verify that if the script is using only one key, isPassed will not change when the second key changes